### PR TITLE
chore(tests): stress testing data generation scripts

### PR DESCRIPTION
**Pull request by miguel-crespo-fdc**
Ref: SRX-YNEAHU

### CODE REVIEW

File: infrastructure/scripts/create-testdata/create-environments.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ testData=${1:-"./testdata_template/environments"}
 for filename in "$testData"/*; do
   configFile="$filename"/config.json
   env=$(basename -- "$filename")
+  env=$(echo "$env" | awk '{print tolower($0)}')
```
Comment (miguel-crespo-fdc) [inline infrastructure/scripts/create-testdata/create-environments.sh:15]: Upper case environments are not accepted by kuberpult

File: infrastructure/scripts/create-testdata/stress_testing/create-releases.sh [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+      randomValue=$(head -c 20 /dev/urandom | sha1sum | awk '{print $1}' | head -c 12)
+    cat <<EOF > "${file}"
+---
+apiVersion: v1
+data:
+  TNS_ADMIN: /db-config
+  TESTDB_SID_LOOKUP_FILENAME: /db-config/locations.json
+kind: ConfigMap
+metadata:
+  name: foo-service
```
Comment (sven-urbanski-freiheit-com) [inline infrastructure/scripts/create-testdata/stress_testing/create-releases.sh:102]: this needs to be unique for the cluster/app combination.
Something like this:
```suggestion
  name: foo-service-$app-$env
```

File: infrastructure/scripts/create-testdata/stress_testing/create-releases.sh [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+  namespace: md
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: foo-service
```
Comment (sven-urbanski-freiheit-com) [inline infrastructure/scripts/create-testdata/stress_testing/create-releases.sh:108]: same here

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

File: infrastructure/scripts/create-testdata/stress_testing/create-releases.sh [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+spec:
+  hosts:
+  - foo-service-$app-$env
+  http:
+  - route:
+    - destination:
+        host: foo-service-$app-$env
+        port:
+          number: 50051
+  random: "${randomValue}"
```
Comment (sven-urbanski-freiheit-com) [inline infrastructure/scripts/create-testdata/stress_testing/create-releases.sh:355]:
```suggestion
```
virtual services don't have a "random" field

Comment (sven-urbanski-freiheit-com) [reply infrastructure/scripts/create-testdata/stress_testing/create-releases.sh:355]: alternatively, remove the entire VirtualService

**Pull request merged**